content/id/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -1047,65 +1047,65 @@ Semua Pod yang ada dimatikan sebelum yang baru dibuat ketika nilai `.spec.strate
 
 #### Membarui Deployment secara Bergulir
 
-Deployment membarui Pod secara [bergulir](/id/docs/tasks/run-application/rolling-update-replication-controller/)
```

Comment:
I think this pointing to a different link? @irvifa 

But anyway, on /en page, there is no link anymore for this sentence.
I suggest removing the link to match with the /en

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#rolling-back-a-deployment

Reply:
Unfortunately I can't find a new link for this, as for your suggestion I already included it on my latest commit.

Reply:
I think there is no changes for this line
will put the code suggestion